docs/src/man/basics.md

```diff
@@ -1565,40 +1565,30 @@ julia> german[Not(5), r"S"]
                 984 rows omitted
 ```
 
-## Basic Usage of Transformation Functions
+## Basic Usage of Manipulation Functions
 
-In DataFrames.jl we have five functions that we can be used to perform
-transformations of columns of a data frame:
+In DataFrames.jl there are seven functions
```

Comment:
As commented above - maybe add one intro paragraph explaining the purpose and teaching approach of this material. Thank you!